README.md

```diff
@@ -4,6 +4,25 @@ This is a Confluence plugin that provides a confidentiality setting for each pag
 
 ![A screenshot of the plugin in action](https://i.imgur.com/w0T7zKF.png)
 
+# Space confidentiality configuration
+By default plugin is not enabled for any space.
+One have to enable it by going to either: Space Tools -> Addons, or Space menu -> Confidentiality
+
+![Space menu](space_menu.png)
```

Comment:
This will be uploaded to the root folder? I'd favor a sub folder or hosting it elsewhere (for example Imgur).